homeassistant/components/device_tracker/owntracks.py

```diff
@@ -78,6 +91,16 @@ def owntracks_event_update(topic, payload, qos):
         if not isinstance(data, dict) or data.get('_type') != 'transition':
             return
 
+        # Check for GPS accuracy
```

Comment:
Owntracks will only send a single 'enter' and 'leave' event - so you can't ignore them.

It doesn't make sense to check accuracy during an 'enter' event in any case, because Owntracks already made the decision - and HA just does as it's told.

We could check accuracy when switching back to gps after processing a leave event, but it's difficult to do correctly since we need to exit the region.

I think I'd suggest not worrying about accuracy when using owntracks regions.